pkg/storaged/mdraid-details.jsx

```diff
@@ -265,10 +265,6 @@ export class MDRaidDetails extends React.Component {
             return mdraid.RequestSyncAction("repair", {});
         }
 
-        function stop_scrub() {
-            return mdraid.RequestSyncAction("idle", {});
```

Comment:
Was it intended to entirely drop the "stop scrubbing" functionality? Couldn't this behave like the Start/Stop action, depending on whether a scrub is already running (that should be in the `SyncAction` property)? FTR, I can't test this - trying to scrub just fails with a [permission error](http://piware.de/tmp/scrub.png) for me, and this isn't exposed in gnome-disks. When I try to call it in d-feet, I get

    "Error requesting 'repair' action on RAID array '/dev/md127': Failed to set requested sync action.Failed to write 'repair' to file '/sys/class/block/md127/md/sync_action': Permission denied (2)".

This error message is confusing as the file doesn't actually exist at all. But this is at the udisks level, so not dependent on this PR.

Reply:
> Was it intended to entirely drop the "stop scrubbing" functionality? 

No, stopping was meant to be done by cancelling the "Repair" job that appears.  But as we found out, there are too many issues with scrubbing to have the button be this prominent now.